client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/MiscellaneousDocumentationIT.java

```diff
@@ -31,20 +31,10 @@
 import java.io.IOException;
 
 /**
- * This class is used to generate the Java Main API documentation.
- * You need to wrap your code between two tags like:
- * // tag::example[]
- * // end::example[]
- *
- * Where example is your tag name.
- *
- * Then in the documentation, you can extract what is between tag and end tags with
- * ["source","java",subs="attributes,callouts,macros"]
- * --------------------------------------------------
- * include-tagged::{doc-tests}/MainDocumentationIT.java[example]
- * --------------------------------------------------
+ * Documentation for miscellaneous APIs in the high level java client.
+ * Code wrapped in {@code tag} and {@code end} tags is included in the docs.
  */
-public class MainDocumentationIT extends ESRestHighLevelClientTestCase {
+public class MiscellaneousDocumentationIT extends ESRestHighLevelClientTestCase {
```

Comment:
I renamed this because I'd like to stick the other miscellaneous methods into this test.

Reply:
++